ddpui/core/orguserfunctions.py

```diff
@@ -275,6 +277,86 @@ def invite_user(orguser: OrgUser, payload: InvitationSchema):
     return payload, None
 
 
+def invite_user_v1(orguser: OrgUser, payload: NewInvitationSchema):
+    """invite a user to an org"""
+    frontend_url = os.getenv("FRONTEND_URL")
+
+    if orguser.org is None:
+        return None, "create an organization first"
+
+    invited_email = payload.invited_email.lower().strip()
+    if OrgUser.objects.filter(
+        org=orguser.org, user__email__iexact=invited_email
+    ).exists():
+        return None, "user already has an account"
+
+    invited_role = Role.objects.filter(uuid=payload.invited_role_uuid).first()
+    if not invited_role:
+        return None, "Invalid role"
+
+    # user can only invite a role equal or lower to their role
+    if invited_role.level > orguser.new_role.level:
```

Comment:
the proper check is whether the inviter has the correct permission to invite another user

Reply:
ok i see you're checking it in the middleware... in that case you can just remove this check

Reply:
I think this should still be there.
 Even though the user has permissions to invite other users, an `account-manager` should not be able to invite a `super-user`.

Reply:
oh like that... hmm all right